setzer/document/document_controller.py

```diff
@@ -93,7 +93,13 @@ def on_keypress(self, widget, event, data=None):
 
     def save_date_loop(self):
         if self.document.filename == None: return True
-        if self.document.save_date <= os.path.getmtime(self.document.filename) - 0.001:
+        if self.document.deleted: return True
```

Comment:
What is this variable for? I think you can remove it. Also in the Document class.

Reply:
It's to avoid an infinite loop of Dialogs :P!

Reply:
I see. Can you rename it to something like "deleted_on_disk_dialog_shown_after_last_save"?

Reply:
I renamed the variable to deleted_on_disk_dialog_shown. I thought that the "after_last_save" would make it a bit too long. Is it fine like that?

Reply:
I think to long name is better tbh, because it makes it obvious what this variable is.

Reply:
You've got a point. I just renamed it to "deleted_on_disk_dialog_shown_after_last_save"!